doc/administrator/mapfile.rst

```diff
@@ -221,6 +221,23 @@ The ``${MAPSERVER_DATA_SUBSELECT}`` is defined as follows:
 The ``${mapfile_data_subselect}`` variable is defined in the ``CONST_vars.yaml``
 configuration file. Its goal is to simplify the writing of the mapfile.
 
+It is required to have the following in the ``VALIDATION`` section of the ``LAYER``:
+
+.. code::
+
+   VALIDATION
+     # For secured layers
+     "default_role_ids" ""
+     # Or
+     # "default_role_ids" "<id of the anonymous role>"
+     "role_ids" "^-?[0-9,]*$"
+   END
+
+The ``default_role_ids`` define the restrictions when mapserver is called without any ``roles_ids``.
+In this situation, if you have set ``default_role_ids`` to an empty value for this layer, then there are
```

Comment:
then there are -> then there is

(But I have a question: does this make sense, is there a use case where you would want to define a layer without any access? Or do I understand the consequences wrongly?)

Reply:
You will have no right if you call MapServer directly without passing throw the proxy, without any ``role_ids``